packages/components/Button/src/Button.types.ts

```diff
@@ -27,6 +27,10 @@ export interface IButtonInfo extends IPressableState {
    * Button text.
    */
   content?: boolean;
+  /**
+   * Right Button icon.
+   */
+   trailingIcon?: boolean;
```

Comment:
Nit, update the comment as well to use trailing.

Reply:
I talked to @PPatBoyd what a more appropriate name for here and he said start/end seems to be canonical for React. So I think the cross platform appropriate name would be `endIcon`? Any thoughts?

Reply:
changed the name to endIcon